src/main/java/org/apache/accumulo/testing/continuous/ContinuousWalk.java

```diff
@@ -54,10 +55,13 @@ public static void main(String[] args) throws Exception {
       ArrayList<Value> values = new ArrayList<>();
 
       int sleepTime = Integer.parseInt(env.getTestProperty(TestProps.CI_WALKER_SLEEP_MS));
+      ConsistencyLevel cl = TestProps
+          .getScanConsistencyLevel(env.getTestProperty(TestProps.CI_BW_CONSISTENCY_LEVEL));
```

Comment:
Should this use `CI_WALKER_CONSISTENCY_LEVEL` instead of `CI_BW_CONSISTENCY_LEVEL`?

Reply:
yep, good catch. Thanks!